metabase/metabase
# Update assets related to Models (previously "Datasets")

### Status
PENDING REVIEW

### What does this PR accomplish?
Updates SVG icons and one SVG illustration that were previously named and related to datasets. They are now referenced as models, as explained in https://github.com/metabase/metabase/issues/19668

### How to test?
- All Models icons should be there
    - action bar
    - collections
    - search page
![image](https://user-images.githubusercontent.com/31325167/149531177-4d608a68-eee7-46f6-b507-7f11bdc61fae.png)
![image](https://user-images.githubusercontent.com/31325167/149531475-c80b0df9-6e8e-4aeb-9463-b560d2106b09.png)
![image](https://user-images.githubusercontent.com/31325167/149531574-d45df916-92a0-4b5e-9109-7b0b3d90d7b9.png)



- When you click on "Turn this into a model", the modal that opens should still show SVG illustration
![image](https://user-images.githubusercontent.com/31325167/149530533-2a783ed5-56d7-43c1-9fe3-f1b9abd68e20.png)
